modules/tensor_mechanics/test/tests/ad_finite_elastic/finite_elastic-noad.i

```diff
@@ -0,0 +1,106 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 3
+  nx = 3
+  ny = 3
+  nz = 3
+[]
+
+[GlobalParams]
+  displacements = 'disp_x disp_y disp_z'
+[]
+
+[Variables]
+  [./disp_x]
+  [../]
+  [./disp_y]
+  [../]
+  [./disp_z]
+  [../]
+[]
+
+[Kernels]
+  [./stress_x]
+    type = StressDivergenceTensors
+    component = 0
+    variable = disp_x
+    # use_displaced_mesh = true
```

Comment:
Are these comments meant to indicate anything to a user? When should `use_displaced_mesh = true` be used in solid mechanics? I know this has been on the list before...I'm being lazy

Reply:
It should be run on the displaced mesh, but the AD version does not work yet as it needs your AD JxW (maybe?).

Reply:
Yessir. Got it